### PR TITLE
add the ability to add a deployment nodeselector

### PR DESCRIPTION
**Pull request by jurgenweber**

### CODE REVIEW

File: argus/values.yaml [hunk 1]
```diff
@@ -8,4 +8,6 @@ collectorVersion: 24200
 disableAlerting: false
 deleteDevices: false
 debug: false
-enableRBAC: true
+deployment:
```
Comment (andrewrynhard) [inline argus/values.yaml]: The default generated by Helm 2.5.1 is 
```
    {{- if .Values.nodeSelector }}
      nodeSelector:
{{ toYaml .Values.nodeSelector | indent 8 }}
    {{- end }}
```
I would prefer to skip the `.deployment` and just use `.Values.nodeSelector`. While I'm not a huge fan of the official helm style, the docs do recommend a flat style: https://docs.helm.sh/chart_best_practices/#flat-or-nested-values

Comment (jurgenweber) [reply argus/values.yaml]: yeah, intresting to me I would nest it depending on file, blah blah. Works for me, it is my first edit to a helm chart.

Comment (andrewrynhard) [reply argus/values.yaml]: I would prefer it too to be quite honest. :) but that is what the community decided on, so I figured we go along with it.

**Comment by andrewrynhard** [comment]
@jurgenweber Ahh, I forgot to mention that you should run a `make` in the root of the repo to generate the tarball. I can fix in a follow up. But this is merged now! Thanks!

**Pull request merged**